.github/workflows/scala.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        scala-version: [ 2.12.11, 2.13.7 ]
+        scala-version: [ 2.12.10, 2.13.7 ]
```

Comment:
why are we downgrading to 2.12.10?

Reply:
Because Scala 2.12.10 is the default Scala shipped by Apache Spark.

Reply:
There are little difference between 2.12.10 and 2.12.11.

For a Spark Cluster (3.1.2), scala-library-2.12.10 and scala-reflect-2.12.11. That's why I downgrade to Scala 2.12.10.

Using Scala 2.12.10, we could find the real Scala library we are using actually during runtime.